force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,9 +236,16 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
+         
+        If (minutes == null)
+         {
+            return null;
+         }
+         final Integer MILLISECONDS_PER_MINUTE;
```

Comment:
What you're doing here works great!
For practice: 
The goal of this one was to make the MILLISECONDS_PER_MINUTE the constant
so something like  'static final Integer MILLISECONDS_PER_MINUTE = 60000'
By making the variable a constant, it doesn't change and can be removed from the method altogether and used elsewhere.